curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/650048b0764f9c1b798200e2.md

```diff
@@ -0,0 +1,392 @@
+---
+id: 650048b0764f9c1b798200e2
+title: Step 57
+challengeType: 0
+dashedName: step-57
+---
+
+# --description--
+
+If you add, update, or remove a task, it should reflect in the UI, but that's not happening now. That's because you're yet to retrieve the tasks. To do this, you need to modify your initial `taskData` set to an empty array.
```

Comment:
```suggestion
If you add, update, or remove a task, it should reflect in the UI. However, that's not happening now because you have yet to retrieve the tasks. To do this, you need to modify your initial `taskData` to be an empty array.
```